requirements.txt

```diff
@@ -15,6 +15,7 @@ flask_wtf==0.14.2
 google-auth==1.7.0
 google_auth_oauthlib==0.4.1
 gunicorn==19.10.0  # Note: Last version to support py2
+httmock==1.3.0
```

Comment:
since you are adding a requirement here, can you also add it into the config/dpkg into the debian packages so that all end to end tests will survive